subservers/taproot-assets.go

```diff
@@ -239,3 +254,13 @@ func (t *taprootAssetsSubServer) WhiteListedURLs() map[string]struct{} {
 		t.cfg.RpcConf.AllowPublicStats || t.remote,
 	)
 }
+
+// Impl returns the actual implementation of the sub-server. This might not be
+// set if the sub-server is running in remote mode.
+func (t *taprootAssetsSubServer) Impl() fn.Option[any] {
+	if t.Server == nil {
+		return fn.None[any]()
```

Comment:
It can _run_ in remote mode, yes. But then the taproot asset channel functionality won't be enabled (for that both `lnd` and `tapd` must be running in integrated mode).